### PR TITLE
HARVESTER: fix vm paused state message tip

### PR DESCRIPTION
**Pull request by WuJun2016**
# links
- https://github.com/harvester/harvester/issues/1987

### CODE REVIEW

File: models/harvester/kubevirt.io.virtualmachine.js [hunk 1]
```diff
@@ -799,12 +801,22 @@ export default class VirtVm extends SteveModel {
...
+    const ignoreConditions = [{
+      name:    'unavailable',
+      error:   false,
+      vmState: this.actualState === PAUSED
+    }];
+
+    const state = this.metadata?.state;
 
-    return super.stateDescription;
+    return ignoreConditions.find(condition => condition.name === state?.name && condition.error === state?.error & condition.vmState) ||
```
Comment (n313893254) [inline models/harvester/kubevirt.io.virtualmachine.js:813]: Tiny tips: Use && not &

File: models/harvester/kubevirt.io.virtualmachine.js [hunk 2]
```diff
@@ -72,6 +72,8 @@ const VMIPhase = {
   Unknown:    'Unknown',
 };
 
+const ignoreMessages = ['pod has unbound immediate PersistentVolumeClaims'];
```
Comment (n313893254) [inline models/harvester/kubevirt.io.virtualmachine.js:75]: Tiny tips: Global variable use uppercase

**Review by n313893254 — COMMENTED** [review]
LGTM

File: models/harvester/kubevirt.io.virtualmachine.js (hunk 1 above)
Comment (WuJun2016) [reply models/harvester/kubevirt.io.virtualmachine.js]: Done

File: models/harvester/kubevirt.io.virtualmachine.js (hunk 2 above)
Comment (WuJun2016) [reply models/harvester/kubevirt.io.virtualmachine.js]: Done

**Comment by WuJun2016** [comment]
PTAL @n313893254

**Review by n313893254 — APPROVED** [review]
LGTM

**Pull request merged**